ci-operator/step-registry/openshift-qe/cluster-density-v2/openshift-qe-cluster-density-v2-commands.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+set -o errexit
+set -o nounset
+set -o pipefail
+set -x
+cat /etc/os-release
+oc config view
+oc projects
+python --version
+pushd /tmp
+python -m virtualenv ./venv_qe
+source ./venv_qe/bin/activate
+
+ES_PASSWORD=$(cat "/secret/password")
+ES_USERNAME=$(cat "/secret/username")
+
+export GSHEET_KEY_LOCATION="${CLUSTER_PROFILE_DIR}/gce.json"
+
+export EMAIL_ID_FOR_RESULTS_SHEET=prubenda@redhat.com
+
+git clone https://github.com/cloud-bulldozer/e2e-benchmarking
+pushd e2e-benchmarking/workloads/kube-burner-ocp-wrapper
+export ITERATIONS=27
```

Comment:
Should we calculate the iterations here based on the size of the cluster?

Reply:
totally agree this should be calculated not hard coded